src/components/CENTRAL_HUB.js

```diff
@@ -1 +1,35 @@
-TODO: create functional component that retrieves crypto info from coingecko
\ No newline at end of file
+import React from 'react'
+import { useState, useEffect } from 'react'
+import axios from 'axios'
+import Coin from './Coin.js'
+
+function CENTRAL_HUB() {
+
+const [coins, setCoins] = useState([])
+
+useEffect(() => {
+    axios
+    .get('https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=100&page=1&sparkline=false')
```

Comment:
Set URL to a constant
```suggestion
    .get(API_URL)
```